src/interner.rs

```diff
@@ -263,6 +281,16 @@ impl<T: Eq + Hash + ?Sized, H: BuildHasher> Interner<T, H> {
         }
     }
 
+    #[cfg(feature = "static")]
+    /// Create an empty interner which will use the given hasher to hash the strings.
+    ///
+    /// The interner is also created with the default capacity.
+    pub const fn with_hasher(hasher: H) -> Self {
+        Interner {
+            arena: RwLock::new(HashMap::with_hasher(hasher)),
```

Comment:
I say just wait for 1.63, since it's only 10 days away and I won't really be able to publish this until then.

The MSRV policy of this crate is an unstated latest stable since I'm not testing anything else. I should add a note to that effect.

Reply:
That sounds good to me! Should there be a `static` feature that enables `hashbrown` like it is now, or should the const method just be switched in if `hashbrown` is enabled? In the second variant, the feature check could simply be removed once the standard library `HashMap` also exposes the const `HashMap::from_hasher`.

Reply:
I think just key it off of `hashbrown`, tbh.

Reply:
Resolved in f5256ca422e2196876a436936fff74db0e9fe05d